static/assets/views/entities.html

```diff
@@ -0,0 +1,43 @@
+
+<h1>{{ctrl.title}} Entity Reference</h1>
+
+<p>
+    This topic describes the entity and type representations for common items that the {{ctrl.title}} API uses. To learn
+    about how to set up your developer account and work with the SKY API, see the
+    <a ng-href="{{ ctrl.getStartedUrl }}">Getting Started</a> guide.
+</p>
+
+<p ng-show="ctrl.showErrorMessage" class="alert alert-danger">
```

Comment:
Yeah, I was thinking that if they contacted us, then we might need a little more info to know how to resolve it (i.e. status code or the response message). Not a big deal. Just a thought...